http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/service/MockKubernetesService.scala

```diff
@@ -0,0 +1,31 @@
+package org.broadinstitute.dsde.workbench.leonardo
+package service
+
+import cats.effect.IO
+import cats.mtl.ApplicativeAsk
+import KubernetesTestData._
+import org.broadinstitute.dsde.workbench.leonardo.http.service.{CreateAppRequest, GetAppResponse, ListAppResponse}
+import org.broadinstitute.dsde.workbench.model.UserInfo
+import org.broadinstitute.dsde.workbench.model.google.GoogleProject
+
+object MockKubernetesServiceInterp extends KubernetesService[IO] {
+  override def createApp(userInfo: UserInfo, googleProject: GoogleProject, appName: AppName, req: CreateAppRequest)(
+    implicit as: ApplicativeAsk[IO, AppContext]
+  ): IO[Unit] =
+    IO.unit
+
+  override def getApp(userInfo: UserInfo, googleProject: GoogleProject, appName: AppName)(
+    implicit as: ApplicativeAsk[IO, AppContext]
+  ): IO[GetAppResponse] =
+    IO(getAppResponse)
```

Comment:
minor...use `IO.pure` to be explicit that `getAppResponse` is pure....`IO()` is lazy, which is unnecessary here